addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -1120,6 +1120,7 @@ var FieldX2Many = AbstractField.extend({
             });
         });
         $.when.apply($, defs).then(function () {
+            event.stopPropagation();
```

Comment:
does not look right. the event.stopPropagation should be somewhere up there, out of the deferred, for example in line 1108.  Here, if the deferred is asynchronous, then it will be too late to stop the propagation, the event will have already passed to its parent